parent/pom.xml

```diff
@@ -227,9 +227,9 @@
         <cxf.jexl.version>2.1.1</cxf.jexl.version>
         <cxf.htrace.version>4.2.0-incubating</cxf.htrace.version>
         <cxf.zest.version>2.1</cxf.zest.version>
-        <cxf.brave.version>4.6.0</cxf.brave.version>
-        <cxf.brave.zipkin.version>1.31.1</cxf.brave.zipkin.version>
-        <cxf.brave.reporter.version>1.0.2</cxf.brave.reporter.version>
+        <cxf.brave.version>5.1.5-SNAPSHOT</cxf.brave.version>
```

Comment:
this will change to 5.1.6 once someone clicks approve on openzipkin/brave#743

Reply:
Approved but @cschneider  :) (sorry was late)